<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterWidget</class>
 <widget class="QWidget" name="FilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>381</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tvFilterFW">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>341</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblFilterSizeFW">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter Size:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbFilterSizeFW">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="cbNFilterFW">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Normalize Filter</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbNOutputFW">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>410</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Normalize Output</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
